index.html

```diff
@@ -3,6 +3,7 @@
 <head>
 	<meta charset="utf-8">
 	<title>Gradplan Graphic</title>
+	<link rel="icon" href="path.ico"
```

Comment:
The > is missing. It would be better if you named the icon favicon.ico so browsers find it automatically as well.